bifrost/release.go

```diff
@@ -291,6 +313,11 @@ func (r *Release) RootLockPath() *string {
 	return &s
 }
 
+func (r *Release) UserLockPath() *string {
```

Comment:
I think the lock should live on the `RootDir` because it will lock all deploys for a project/config

Reply:
Oh oops, good catch; copy paste error